schema/deploy/computed_columns/project_manager_names_concatenated.sql

```diff
@@ -0,0 +1,31 @@
+-- Deploy cif:computed_columns/project_manager_names_concatenated to pg
+
+-- instead returns text, concatenated, it'll be an empty string if there are no managers. Then it wouldn't need to take an argument, filter for empty string and not null (check filters for null or empty options). includes insensitive empty string (potentially consider refactoring to use tvector later)
+
+begin;
+
+create or replace function cif.project_manager_names_concatenated(cif.project)
+returns text
+as $$
+declare
+  index int;
+  names text := '';
+begin
+  for index in 1..4
+  loop
+    select concat(names, (select cif.cif_user_full_name(
+            (select row(cif_user.*)::cif.cif_user from cif.cif_user
+            where cif.cif_user.id =
+                (select cif_user_id from cif.project_manager
+                where cif.project_manager.project_id=$1.id
+                and cif.project_manager.project_manager_label_id=index))))::text);
```

Comment:
We only want primary managers here. I'd suggest filtering on the id's where the label name contains `primary`
```suggestion
                and cif.project_manager.project_manager_label_id in (select id from cif.project_manager_label where label ilike '%primary%')))))::text);
```